### ISSUE TITLE
Cannot export course on k8s in v16.0.3

### ISSUE THREAD

**Issue by zameel7** [overhangio/tutor-minio#33]
Tutor version: 16.1.8
Tutor-minio version: 16.0.3

After enabling the plugin, the course import and export shows error.

<img width="931" alt="image" src="https://github.com/overhangio/tutor-minio/assets/43750093/6bd0b083-6c16-4dcd-a9a1-6d9f6c65c961">

While checking the previous commits and releases, I saw the issue raised for S3 bucket related for dev. Hence I checked using the 16.0.2 version for the same tutor version and seems to solve the problem. 

<img width="931" alt="image" src="https://github.com/overhangio/tutor-minio/assets/43750093/4974babf-fa1a-434b-af35-edbd3e527223">

Is the version 16.0.2 the right version to be used for Palm?

**Comment by regisb** [comment]
Are you running tutor in dev or local mode? What are the error logs coming from the cms and cms-worker containers?

**Comment by zameel7** [comment]
We are running tutor on k8s. I saw some error logs thrown from somewhere in
this function:
https://github.com/openedx/edx-platform/blob/cf6e266f314cf71a6587a42b7dc0831da0226d84/cms/djangoapps/contentstore/tasks.py#L302

**Comment by regisb** [comment]
Can you be more specific?

**Comment by zameel7** [comment]
> Can you be more specific?

Let me reproduce the issue and get some logs for you.

**Comment by zameel7** [comment]
```
[2024-02-06 02:05:15,867: ERROR/ForkPoolWorker-1] cms.djangoapps.contentstore.tasks.export_olx[aa43aece-5369-47a1-9f5e-9dad8678f162]: Error exporting course course-v1:blend_ed_staging+BL102D+2023_SEP
Traceback (most recent call last):
  File "/openedx/edx-platform/cms/djangoapps/contentstore/tasks.py", line 329, in export_olx
    artifact.file.save(name=os.path.basename(tarball.name), content=File(tarball))
  File "/openedx/venv/lib/python3.8/site-packages/django/db/models/fields/files.py", line 89, in save
    self.name = self.storage.save(name, content, max_length=self.field.max_length)
  File "/openedx/venv/lib/python3.8/site-packages/django/core/files/storage.py", line 54, in save
    name = self._save(name, content)
  File "/openedx/venv/lib/python3.8/site-packages/storages/backends/s3boto.py", line 425, in _save
    key = self.bucket.get_key(encoded_name)
  File "/openedx/venv/lib/python3.8/site-packages/boto/s3/bucket.py", line 193, in get_key
    key, resp = self._get_key_internal(key_name, headers, query_args_l)
  File "/openedx/venv/lib/python3.8/site-packages/boto/s3/bucket.py", line 230, in _get_key_internal
    raise self.connection.provider.storage_response_error(
boto.exception.S3ResponseError: S3ResponseError: 403 Forbidden

[2024-02-06 02:05:15,990: INFO/MainProcess] Task cms.djangoapps.cms_user_tasks.tasks.send_task_complete_email[157e845d-f670-4de3-aefe-973a517807eb] received
```

@regisb , this is the logs I get in cms-worker

**Comment by regisb** [comment]
I really do not understand why the change you propose in #34 can address this issue. There is a reason why the `MinIOStorage` was removed, and you can't ignore that reason to put it back in. You need to figure out why there is a 403 error and address that using the default S3Boto3Storage backend.

**Comment by FahadKhalid210** [comment]
@zameel7 closing this issue as it's fixed now and new release([v16.0.4](https://github.com/overhangio/tutor-minio/releases/tag/v16.0.4)) is published.
